Sources/Complex/Complex.swift

```diff
@@ -472,26 +472,39 @@ extension Complex {
     (length, phase)
   }
   
-  /// Constructs a complex value from polar coordinates.
+  /// A complex value specified with  polar coordinates.
```

Comment:
I think initializer documentation usually starts with `Creates a/an`.
```suggestion
  /// Creates a complex value specified with polar coordinates.
```